engine/src/cubos/engine/assets/assets.cpp

```diff
@@ -600,4 +605,18 @@ std::vector<AnyAsset> Assets::listAll() const
         out.emplace_back(entry);
     }
     return out;
+}
+
+std::type_index Assets::type(const AnyAsset& handle) const
+{
+    auto assetEntry = this->entry(handle);
+    CUBOS_ASSERT(assetEntry != nullptr, "Could not find asset's type: No entry found!");
+    if (assetEntry->status == Status::Loaded)
+    {
+        return assetEntry->type;
+    }
+
+    auto bridge = this->bridge(handle);
+    CUBOS_ASSERT(bridge != nullptr, "Could not find asset's type: No bridge found!");
```

Comment:
```suggestion
    CUBOS_ASSERT(bridge != nullptr, "Could not find asset's type");
```

Turns out `bridge()` already logs the error reason anyway